bundles/org.openhab.binding.onecta/src/main/java/org/openhab/binding/onecta/internal/handler/OnectaBridgeHandler.java

```diff
@@ -115,41 +113,50 @@ public void dispose() {
 
     private void pollDevices() {
         logger.debug("pollDevices.");
-        if (onectaConnectionClient.isOnline()) {
-            updateStatus(ThingStatus.ONLINE);
 
-        } else {
-            if (getThing().getStatus() != ThingStatus.OFFLINE) {
-                updateStatus(ThingStatus.OFFLINE);
+        if (getThing().getStatus().equals(ThingStatus.OFFLINE)) {
+            try {
+                logger.debug("Try to restore connection ");
+                onectaConnectionClient.restoreConnecton();
+
+                if (onectaConnectionClient.isOnline()) {
+                    updateStatus(ThingStatus.ONLINE);
+                }
+            } catch (DaikinCommunicationException e) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
+                        "Try to restore connection. See log for more information. ");
```

Comment:
Proposal: `Trying to restore connection. See log for more information.` <-- note that there is an additional space in your message at the end.

Also, messages displayed on the UI should be externalized to a `.properties` file in order to be translatable. Put them into a file located at `src/main/resources/OH-INF/i18n/onecta.properties`. Instead of a literal string you can pass `@text/my.property.key` here, which will look up `my.property.key` in the i18n file.